phpstan-baseline.neon

```
@@ -71965,3 +71965,75 @@ parameters:
 			message: "#^Call to an undefined method MauticPlugin\\\\MauticTagManagerBundle\\\\Security\\\\Permissions\\\\TagManagerPermissions\\:\\:expects\\(\\)\\.$#"
 			count: 1
 			path: plugins/MauticTagManagerBundle/Tests/Unit/Security/Permissions/TagManagerPermissionsTest.php
+
+		- '#\$id type mapping mismatch: database can contain string but property expects int.#'
```

Was this line added manually? I fear it would cause trouble when we re-generate the baseline in the future.

---

In this [commit](https://github.com/mautic/mautic/pull/12262/commits/ad66b80d6cd867e0bdd78bb9ce57b68a990305d7) I did not ignore any error, I reset to what we have in `5.x`